graylog2-server/src/test/java/org/graylog/testing/completebackend/GraylogBackendExtension.java

```diff
@@ -44,6 +45,16 @@ public Object resolveParameter(ParameterContext parameterContext, ExtensionConte
             return ContainerMatrixHierarchicalTestExecutor.requestSpecification.orElse(null);
         } else if (paramType.equals(SearchServerInstance.class)) {
             return ContainerMatrixHierarchicalTestExecutor.graylogBackend.map(GraylogBackend::searchServerInstance).orElse(null);
+        } else if (paramType.equals(GraylogApis.class)) {
+            if (ContainerMatrixHierarchicalTestExecutor.requestSpecification.isPresent() && ContainerMatrixHierarchicalTestExecutor.graylogBackend.isPresent()) {
+                return new GraylogApis(
+                        ContainerMatrixHierarchicalTestExecutor.requestSpecification.get(),
+                        ContainerMatrixHierarchicalTestExecutor.graylogBackend.get()
+                );
+            } else {
+                System.out.println("Failed to resolve GraylogApis");
```

Comment:
LOG.error or sth similar? returning `null` seems ok, the following test will probably fail anyway - and with an NPE but I think logging as usual seems reasonable?

Reply:
Why not just throw an exception to make this fail as hard as possible?

Reply:
My fault, was a debugging test on my side. Removed and throwing exception now. Thanks!